### ISSUE TITLE
issues in mot.py

### ISSUE THREAD

**Issue by ronghuaUTTfr** [shijieS/SST#17]
very nice work, but I'd like to say there are some mistakes with your code. In your original code, the images and the bounding boxes are not corresponding. 
mot.py, line 113: `current = self.recorder[frame_index]` 
should be: `current = self.recorder[frame_index-1]`
mot.py, line 152: `next = self.recorder[next_frame_index]` 
should be: `next = self.recorder[next_frame_index-1]`
OR:
mot.py, line 99: 
`image_path = os.path.join(self.folder, 'img1/{0:06}.jpg'.format(frame_index))`
modified version:
 `image_path = os.path.join(self.folder, 'img1/{0:06}.jpg'.format(frame_index+1))`

**Comment by wangxianrui** [comment]
I agree with you, have you tried it?

**Comment by ronghuautt** [comment]
> I agree with you, have you tried it?
Indeed, the modification has no obvious influence on the final results (I did not fully retrain the network, maybe you can do it). Since his program is actually learning the background features, a portion of a misaligning can be ignored (perhaps, I guess).
